xml/decoders/nmra/accelDecel.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Just checking. For a number of files in this PR, GitHub tells that this line is changed. But I don't see how.

Reply:
Me neither.  Pulling out the line and doing a `cmp` didn't find anything.